R/derive_param_response.R

```diff
@@ -0,0 +1,248 @@
+#' Add a Parameter Indicating If a Subject Had a Response
```

Comment:
should we add here "before first PD (Progressive Disease)"?

Reply:
Ok added